tests/unit/deposit/test_api_simpleflow.py

```diff
@@ -146,7 +146,7 @@ def test_simple_rest_flow(mocker, api, api_client, db, es,
     # Not allowed to edit drafts
     response = client.put(
         links['self'], data=json.dumps(test_data), headers=auth_headers)
-    assert response.status_code == 403
+    assert response.status_code == 400
```

Comment:
I think we should keep this test the way it was (i.e. returning a 403 because of the deposit status) by adding the `doi` field in the `test_data` (it should be there anyway)